templates/rhpam70-authoring-ha.yaml

```diff
@@ -365,6 +365,10 @@ parameters:
   name: SSO_DISABLE_SSL_CERTIFICATE_VALIDATION
   value: "false"
   required: false
+- displayName: RH-SSO Principal Attribute
+  description: RH-SSO Principal Attribute to use as username. Defaults to preferred_username
```

Comment:
This environment variable do not have set default value (value: "preferred_username"). From the jira description [CLOUD-2722](https://issues.jboss.org/browse/CLOUD-2722) I would expect, that for an empty value of the SSO_PRINCIPAL_ATTRIBUTE parameter is used as default value "sub". @ruromero please, can you add the default value for all templates?

Reply:
@jakubschwan once the PR for CLOUD-2722 is merged I’ll apply the changes